src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -50,6 +50,7 @@ stop() {
         for cluster in ${CLUSTERS[@]}; do
             ../src/mstop.sh $cluster
         done
+        killall -9 rbd-mirror
```

Comment:
```suggestion
        pids=$(pgrep rbd-mirror)
        if [ -n "$pids" ]; then
            echo Killing rbd-mirror processes: $pids
            kill -9 $pids
        fi
```

Saw @idryomov's suggestion; untested, but ought to work

Reply:
thanks @zmc. updated!

Reply:
This seems to be functionally equivalent to a simple `killall -9`.  With my "it would be nice" comment I meant to arrange to kill only the rbd-mirror daemon instance that was started earlier by the test and no other (in case there are any running).  But, like I said, I think it's fine for the purposes of this script.